.gitignore

```diff
@@ -146,10 +147,22 @@ iphone/Maps/app.omaps/
 
 *.autosave
 
+# CMake/ninja
 cmake-build-*
 build/
+.cmake/
```

Comment:
Looks like the cmake is not used properly. All temporary files should be created in a different directory, not in the source directory. E.g.:

```bash
cd ..
mkdir omim-build-debug
cd omim-build-debug
cmake -DCMAKE_BUILD_TYPE=Debug -GNinja ../organicmaps/
ninja [optional_target]
```

Reply:
Right, I must have messed it up when I was trying to understand the macdeployqt issue, but shouldn't we ignore those anyway? Or is it used for important files?

Reply:
Please remove them, using cmake in the same dir as the source is an anti-pattern.